generated/nidmm/attributes.py

```diff
@@ -52,16 +52,20 @@ def set(self, session, channel, value):
         session._set_attribute_vi_boolean(channel, self._attribute_id, value)
 
 
-class AttributeEnum(AttributeViInt32):
+class AttributeEnum(Attribute):
 
-    def __init__(self, attribute_id, enum_meta_class, channel=''):
+    def __init__(self, underlying_attr_type, enum_meta_class):
```

Comment:
Underlying_attr_type is not the type, it's an actual instance of an attribute. We can rename the parameter / member, or truly pass the type and instantiate the object here.

There's some redundancy here, because AttributeEnum also derives from Attribute, as well as the thing passed in. I think we can live with it, however, even though it's a bit confusing.

Reply:
Changed how AttributeEnum works and is instatiated